system/src/system_network_diagnostics.cpp

```diff
@@ -109,8 +109,7 @@ class NetworkCache
 private:
 #if Wiring_Cellular
     CellularSignal sig_;
-    CellularGlobalIdentity cgi_{.size = sizeof(CellularGlobalIdentity),
-                                .version = CGI_VERSION_LATEST};
+    CellularGlobalIdentity cgi_ = {};
```

Comment:
We're clearing the `.size` and `.version` fields.

Reply:
:+1:

Reply:
Actually, let's leave it as-is. `getCGI()` anyway configures them correctly.